backend/src/main/java/com/soen341/instagram/controller/Login Controller/src/LoginController.java

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Add the annotation @RestController here so spring know this class is a controller